podcasts/podcasts.entitlements

```diff
@@ -17,7 +17,7 @@
 		<string>webcredentials:play.pocketcasts.com</string>
 		<string>applinks:pcast.pocketcasts.net</string>
 		<string>applinks:pca.st</string>
-		<string>applinks:pocketcasts.com</string>
+		<string>applinks:pocketcasts.com?mode=developer</string>
```

Comment:
Temporary change to allow us to test in real time changes in the `apple-app-site-association` file

Reply:
Should this be included in `trunk` or only in PRs for testing?

Reply:
I will revert it now. So this doesn't go in by mistake later on!